universe/Species.cpp

```diff
@@ -146,6 +146,25 @@ std::string Species::Dump() const {
     return retval;
 }
 
+std::string Species::GameplayDescription() const
+{
+    std::string result(UserString(m_gameplay_description));
+
+    for (std::vector<boost::shared_ptr<Effect::EffectsGroup> >::const_iterator it = m_effects.begin();
+         m_effects.end() != it; ++it)
+    {
+        if (it == m_effects.begin())
```

Comment:
However, it could happen that a species has several EffectsGroups assigned but none of them has a description.  This would lead to two newlines.